doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -674,20 +671,15 @@ file is a particular case of module called *library file*.
 
 .. cmd:: Declare ML Module {+ @string }
 
-   This commands loads the OCaml compiled files
-   with names given by the :token:`string` sequence
-   (dynamic link). It is mainly used to load tactics dynamically. The
+   This commands dynamically loads the specified OCaml compiled files.
+   It is mainly used to load tactics dynamically. The
```

Comment:
```suggestion
   It is used to load plugins dynamically. The
```
Most plugins do provide tactics but not all of them.

The description would deserve the following addition:

>This command is reserved to developers of plugins, who should provide a `.v` file containing it. Users of the plugins will generally load the `.v` file.

Reply:
String is a logical paths that gets mapped to a file system path, not a file system path?

Reply:
I don't really know how this command works, but in general people write only the name of their `mllib` file without the extension and without the filesystem location either, and that works. Example: https://github.com/mattam82/Coq-Equations/blob/6a222c52bbb384c788c145b6636fe34d9375af1a/theories/Init.v#L11-L13